src/snowcli/cli/snowpark/procedure_coverage/manager.py

```diff
@@ -48,15 +44,17 @@ def report(
         orig_get_python_source = coverage.python.get_python_source
 
         def new_get_python_source(filename: str):
-            new_filename = filename[filename.index("app.zip/") + len("app.zip/") :]
-            return orig_get_python_source(new_filename)
+            file_path = Path(filename)
+            parts = file_path.parts
+            new_path = Path(artefact_name) / "/".join(
+                parts[parts.index("app.zip") + 1 :]
```

Comment:
Is `app.zip` present here a different `app.zip` than our previous hardcoded artefact name?

Reply:
Good catch, fixed